deeplake/core/tests/test_concurrent.py

```diff
@@ -0,0 +1,44 @@
+import deeplake
+
+
+import numpy as np
+import tqdm
+
+from concurrent.futures import ProcessPoolExecutor
+
+ds_path = "concurrent_test_ds/"
+
+
+def rand_images():
+    return np.random.randint(0, 255, size=(10, 256, 256, 3), dtype=np.uint8)
+
+
+def rand_labels():
+    return np.random.randint(0, 10, size=(10,), dtype=np.uint8)
+
+
+def worker(worker_id: int):
+    # time.sleep(worker_id / 10)
+    ds = deeplake.load(ds_path)
+    ds.checkout("abcd")
+    print(f"Hello from worker {worker_id}!")
+    with ds.concurrent():
+        for i in tqdm.tqdm(range(10)):
+            ds.images.extend(rand_images())
+            ds.labels.extend(rand_labels())
+        return len(ds)
+
+
+def test_concurrent():
```

Comment:
@farizrahman4u 1 test of a basic use case is not nearly enough for such a major feature.